transfers/transfer.py

```diff
@@ -11,12 +11,15 @@
 import base64
 import logging
 import logging.config  # Has to be imported separately
+import models
```

Comment:
I usually separate project-specific import from Python builtin imports, which is why it was further down (sorry, I didn't mention that earlier).  Also, py2/3 compatibility needs this to be `from . import models`